Bridge/Bridge.java

```diff
@@ -0,0 +1,38 @@
+class Bridge{
+	private static int time;
+	private static int[] disc, low;
+	private static boolean[] visited;
+	
+	public static boolean[] bridges(int[][][] graphWithEdgeInfo, int EdgeCount){
```

Comment:
EdgeCount is calculated easily in O(|graphWithEdgeInfo|) which doesn't affect overall complexity. I would like to overload `bridges` like
```
public static boolean[] bridges(int[][][] graphWithEdgeInfo) {
    int m2 = 0;
    for(int[][] row : graphWithEdgeInfo)m2 += row.length;
    return bridges(graphWithEdgeInfo, m2/2);
}
```